5-Heroku/server/public/styles/styles.css

```diff
@@ -0,0 +1,82 @@
+
+:root {
```

Comment:
what is this?

Reply:
this the styles i have from .css file from what i got from the build.

Reply:
yeah same comment as above

Reply:
plz delete public folder from git. build your project when u deploy to heroku